app/views/users/sessions/redirect_from_magic_link.html.erb

```diff
@@ -6,7 +6,7 @@
       <div class="field">
         <%= hidden_field_tag :login_token, @login_token %>
       </div>
-      <%= submit_tag "Continue", class: "govuk-button" %>
+      <%= tag.button("Continue", class: "govuk-button", data: { module: "button" }) %>
```

Comment:
Switch this to `f.govuk_submit` by changing `form_tag` to `form_for` (even if there’s no model object)?

Reply:
I was under the impression `form_for` _needed_ an object to bind to

Reply:
Actually maybe `form_with` is the answer here. Let me test it.

Reply:
> I was under the impression form_for needed an object to bind to

I think that was maybe true once but changed at some point.